src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj

```diff
@@ -16,4 +17,20 @@
     <Compile Include="../Coreclr.TestWrapper/MobileAppHandler.cs" Link="MobileAppHandler.cs" />
   </ItemGroup>
 
+  <Target Name="Test">
+    <!-- This avoids attempting to run tests on this assembly because it is being included in libs.tests.
+         TODO: Build XUnitLogChecker once, in a common location for coreclr and libraries, then zip and upload it once,
+         and then consume the same payload in both the coreclr and libraries tests. -->
+  </Target>
+ 
+  <Target Name="ZipTestArchive" AfterTargets="PrepareForRun">
```

Comment:
Forgot to condition this. It's now failing in mono. Shouldn't run there.

Reply:
A condition isn't required here as the project shouldn't built at all on Mono.

Reply:
Okay, yes. Since I solved the problem in the other [comment ](https://github.com/dotnet/runtime/pull/93906#discussion_r1388496293) (it was due to the property not existing there), it can be removed from here.